pulpcore/cli/ansible/content.py

```diff
@@ -28,4 +50,96 @@ def content(ctx: click.Context, pulp_ctx: PulpContext, content_type: str) -> Non
         raise NotImplementedError()
 
 
-content.add_command(list_command())
+list_options = [
+    pulp_option("--name", help=_("Name of {entity}")),
+    pulp_option("--namespace", help=_("Namespace of {entity}")),
+    pulp_option("--version", help=_("Version of {entity}")),
+    pulp_option(
+        "--only-highest",
```

Comment:
Do you think, `--highest` or `--latest` would be a better name?

Reply:
`--latest` sounds good.